src/index.html

```diff
@@ -4,10 +4,51 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+    <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars">
```

Comment:
tag body can’t have stars class. Can be body, pageBody, or something like that. But try not to add a class for body unless completely necessary
you must implement `6` blocks with stars class